src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
```

Comment:
remove margin

Reply:
Also I cannot find the problem with my tests